app/services/solana_rpc_client.rb

```diff
@@ -1,13 +1,15 @@
 # frozen_string_literal: true
```

Comment:
If we are cleaning this class up we could remove the `default_client` method. So please make it as well.